sycl/include/sycl/backend.hpp

```diff
@@ -127,7 +127,19 @@ auto get_native(const SyclObjectT &Obj)
     throw sycl::runtime_error(errc::backend_mismatch, "Backends mismatch",
                               PI_ERROR_INVALID_OPERATION);
   }
-  return Obj.template get_native<BackendName>();
+  return reinterpret_cast<backend_return_t<BackendName, SyclObjectT>>(
+      Obj.getNative());
+}
+
+template <backend BackendName, bundle_state State>
+auto get_native(const kernel_bundle<State> &Obj)
+    -> backend_return_t<BackendName, kernel_bundle<State>> {
```

Comment:
Do we have style guide for this spelling of the return type? I don't think I've seen it anywhere in the code before...

Reply:
I don't think we do, and, personally, I don't see the point of it here and would rather get rid of it, just did it for consistency with the existing code in this file. I don't mind making the change as part of this PR if you agree.

Reply:
> for consistency with the existing code in this file

Missed that, thanks for the explanation.

> I don't mind making the change as part of this PR if you agree.

I think that should be a standalone PR having no ties to this one (i.e. shouldn't be even required for you).